mobile/gen_bindings.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+
+mkdir -p build
+
+# Check falafel version.
+falafelVersion="0.3"
+falafel=$(which falafel)
+if [ $falafel ]
+then
+        version=$($falafel -v)
+        if [ $version != $falafelVersion ]
+        then
+                echo "falafel version $falafelVersion required"
+                exit 1
+        fi
+        echo "Using plugin $falafel $version"
+else
+        echo "falafel not found"
+        exit 1
+fi
+
+pkg="lndmobile"
+target_pkg="github.com/lightningnetwork/lnd/lnrpc"
+
+# Generate APIs by passing the parsed protos to the falafel plugin.
+opts="package_name=$pkg,target_package=$target_pkg,listeners=lightning=lightningLis walletunlocker=walletUnlockerLis,mem_rpc=1"
+protoc -I/usr/local/include -I. \
```

Comment:
Just thought I'd mention that I didn't have this command, had to run `brew install protobuf` to get it. Might be worth adding it as a prerequisite to the docs.